apps/ledger-live-desktop/src/renderer/families/cosmos/DelegationFlowModal/types.ts

```diff
@@ -18,15 +18,15 @@ export type StepProps = {
   onRetry: (a: void) => void;
   onClose: () => void;
   openModal: OpenModal;
-  optimisticOperation: CosmosOperation | undefined;
+  optimisticOperation: Operation | undefined;
```

Comment:
Why do we have to do this change ? `CosmosOperation` is a more precise type no way to keep it ?

Reply:
I would've kept it indeed, but somehow this is handled at some point by some generic component that doesn't know about `CosmosOperation` ([here](https://github.com/LedgerHQ/ledger-live/blob/d90f58c7b65a67f829627ec2678b96eca7f5934c/apps/ledger-live-desktop/src/renderer/families/cosmos/DelegationFlowModal/Body.tsx#L68))

The full error:
```
apps/ledger-live-desktop/src/renderer/families/cosmos/DelegationFlowModal/Body.tsx:68:5 - error TS2322: Type '({ account, parentAccount, transaction, status, transitionTo, onOperationBroadcasted, onTransactionError, setSigned, onConfirmationHandler, onFailHandler, }: { transitionTo: (a: string) => void; account?: AccountLike | null | undefined; parentAccount?: Account | null | undefined; transaction?: Transaction | null | u...' is not assignable to type 'ComponentType<StepProps>'.
  Type '({ account, parentAccount, transaction, status, transitionTo, onOperationBroadcasted, onTransactionError, setSigned, onConfirmationHandler, onFailHandler, }: { transitionTo: (a: string) => void; account?: AccountLike | null | undefined; parentAccount?: Account | null | undefined; transaction?: Transaction | null | u...' is not assignable to type 'FunctionComponent<StepProps>'.
    Types of parameters '__0' and 'props' are incompatible.
      Type 'PropsWithChildren<StepProps>' is not assignable to type '{ transitionTo: (a: string) => void; account?: AccountLike | null | undefined; parentAccount?: Account | null | undefined; transaction?: Transaction | null | undefined; ... 5 more ...; onFailHandler?: Function | undefined; }'.
        Types of property 'onOperationBroadcasted' are incompatible.
          Type '(a: CosmosOperation) => void' is not assignable to type '(a: Operation) => void'.
            Types of parameters 'a' and 'a' are incompatible.
              Type 'Operation' is not assignable to type 'CosmosOperation'.
                Types of property 'extra' are incompatible.
                  Type 'unknown' is not assignable to type 'CosmosOperationExtra'
```